### PR TITLE
Add Service Cloud web-to-case contact form

### PR DESCRIPTION
**Pull request by DanielBrierton**

### CODE REVIEW

**Comment by DanielBrierton** [comment]
:heavy_check_mark: Deploy Preview for *cdf-website* ready!


:hammer: Explore the source changes: 09a6293803d466502046b48eb56429f43c342a8c

:mag: Inspect the deploy log: [https://app.netlify.com/sites/cdf-website/deploys/60c22fedb58cfa000733c0ea](https://app.netlify.com/sites/cdf-website/deploys/60c22fedb58cfa000733c0ea?utm_source=github&utm_campaign=bot_dl)

:sunglasses: Browse the preview: [https://deploy-preview-87--cdf-website.netlify.app](https://deploy-preview-87--cdf-website.netlify.app?utm_source=github&utm_campaign=bot_dp)

File: pages/contact/index.vue [hunk 1]
```diff
@@ -0,0 +1,603 @@
...
+  .c-form-fieldset__legend {
+    font-size: 1.92rem;
+    -webkit-margin-after: 2.4rem;
+    margin-block-end: 2.4rem;
+    font-weight: 700;
+    margin-top: 0;
+    font-size: 2.4rem;
+    margin-bottom: .8rem;
+    -webkit-margin-after: .8rem;
+    margin-block-end: .8rem;
```
Comment (Jonic) [inline pages/contact/index.vue:430]: Looks like some duplicate styles? Not sure how this has happened! Same for the `min-width: 800px` media query below.

**Review by Jonic — APPROVED** [review]
LGTM? I guess? Those Sauce style being copied/pasted in looks like they've introduced some duplicates, but nothing outside individual selectors, so probably not worth fixing? Is there a reason you couldn't just the mixins and tokens in Sauce? I'm guess Vue/Nuxt doesn't support dart-sass in the version used here?

As for everything else ... I mean yeah it *looks* fine, but in the absence of tests I'm going to have to take you on your word that it does what it's supposed to!

File: pages/contact/index.vue (hunk 1 above)
Comment (DanielBrierton) [reply pages/contact/index.vue]: Good catch. Not sure what happened there exactly. Could've easily been human error.

**Pull request merged**